packages/pn-personafisica-webapp/src/utility/__test__/contacts.utility.test.ts

```diff
@@ -186,6 +202,61 @@ describe('Contacts utility test', () => {
     ]);
   });
 
+  it('test sortAddresses function - legal contacts', () => {
+    const sercqDefaultAddress = digitalAddressesSercq[digitalAddressesSercq.length - 1];
+
+    // test with two default addresses
+    let addresses = [digitalAddresses[0], sercqDefaultAddress];
```

Comment:
Instead to use digitalAddresses[0] do a find in the digitalAddresses searching for PEC and default address